service-api/src/main/java/greencity/security/dto/ownsecurity/OwnSignUpDto.java

```diff
@@ -22,13 +21,9 @@
 @NoArgsConstructor
 @Builder
 public class OwnSignUpDto {
-    @NotBlank
-    @Length(
-        min = ValidationConstants.USERNAME_MIN_LENGTH,
-        max = ValidationConstants.USERNAME_MAX_LENGTH)
```

Comment:
I see that ` UserForListDto, UserManagementDto, UserManagementUpdateDto, UserUpdateDto, EmployeeSignUpDto` use the same validation  
   ```
    min = ValidationConstants.USERNAME_MIN_LENGTH,
    max = ValidationConstants.USERNAME_MAX_LENGTH)
```
Should it also be changed to `regexp = ValidationConstants.USERNAME_REGEXP`?